homeassistant/components/airzone/binary_sensor.py

```diff
@@ -0,0 +1,112 @@
+"""Support for the Airzone sensors."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from dataclasses import dataclass
+from typing import Any, Final
+
+from aioairzone.const import (
+    AZD_AIR_DEMAND,
+    AZD_DEMAND,
+    AZD_ERRORS,
+    AZD_FLOOR_DEMAND,
+    AZD_NAME,
+    AZD_PROBLEMS,
+    AZD_ZONES,
+)
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_PROBLEM,
+    DEVICE_CLASS_RUNNING,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import AirzoneEntity
+from .const import DOMAIN
+from .coordinator import AirzoneUpdateCoordinator
+
+
+@dataclass
+class AirzoneBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """A class that describes airzone binary sensor entities."""
+
+    attributes: dict[str, str] | None = None
+
+
+BINARY_SENSOR_TYPES: Final[tuple[AirzoneBinarySensorEntityDescription, ...]] = (
+    AirzoneBinarySensorEntityDescription(
+        attributes={
+            "air": AZD_AIR_DEMAND,
+            "floor": AZD_FLOOR_DEMAND,
+        },
```

Comment:
What do these attributes represent?

Reply:
"air" represents that the system has enabled the air conditioning system and "floor" represents that the system has enabled the radiators or floor the radiating floor.

Reply:
Usually we would make these separate sensors instead of attributes

Reply:
Ok, I'll split this into 2 sensors then.

Reply:
Done in 0363ba9d70